lib/__tests__/definition.js

```diff
@@ -291,6 +291,36 @@ it("resolves aliases in meta data", done => {
   );
 });
 
+it("DOES NOT resolve aliases in metadata when resolveMetaAliases flag is NOT set (false)", done => {
```

Comment:
Test to ensure not setting the `resolveMetaAliases` option, defaulted to `false` will not resolve aliases in metadata